testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -30,6 +31,21 @@ def retrieve_build_host_id
   visit("https://#{$server.full_hostname}/rhn/manager/cm/build")
 end
 
+Then(/^I wait until the image build "([^"]*)" is completed$/) do |image_name|
+  steps %(
+    When I wait at most 3300 seconds until event "Image Build #{image_name} scheduled by kiwikiwi" is completed
+    And I wait at most 300 seconds until event "Image Inspect 1//#{image_name}:latest scheduled by kiwikiwi" is completed
+  )
+end
+
+Then(/^I am on the image page of "([^"]*)"$/) do |image_name|
+  step %(I navigate to "#{image_name}" page)
```

Comment:
This step is not implemented yet, right? 🤔

Reply:
Right.  Thank you.
Actually, for this step we don't need to pass the name of the image.
I have few ideas, what do you prefer?

1. Just keep the original step:
`And I navigate to "os-images/1/" page`

2. Add a step to hide 'os-images/1/':
`And I am on the ...`

3. Wrap the last 2 steps of the scenario in one step:
`Then the image should exist on server

Reply:
I would try to map somehow the "suse_os_image" into "os-images/1/", so you can then do something like...
```
image_path = get_image_path(image_name)
step %(I navigate to "#{image_path}" page)
```

Reply:
Good idea!
 I have added the function `compute_image_url`